src/utils/nonNull.ts

```diff
@@ -19,8 +19,8 @@ export function nonNullProp<TSource, TKey extends keyof TSource>(
 export function nonNullValue<T>(value: T | undefined, propertyNameOrMessage?: string): T {
   if (isNullOrUndefined(value)) {
     throw new Error(
-      l10n.t('Internal error: Expected value to be neither null nor undefined') +
-        (propertyNameOrMessage ? `: ${propertyNameOrMessage}` : ''),
+      `${errorMessage.internalErrorNullOrUnderined} +
+      ${propertyNameOrMessage ? `: ${propertyNameOrMessage}` : ''}`,
     )
   }
```

Comment:
u can change it like this

```suggestion
export function nonNull<T>(value: T | undefined, propertyNameOrMessage: string): T {
  if (value === undefined || value === undefined) {
    throw new Error(`${errorMessage.internalErrorNullOrUnderined} ${propertyNameOrMessage}`)
  }

  return value
}
```

Reply:
added